ci-operator/config/openshift/verification-tests/openshift-verification-tests-master.yaml

```diff
@@ -0,0 +1,25 @@
+build_root:
+  image_stream_tag:
+    cluster: 'https://api.ci.openshift.org'
+    name: 'bushslicer'
+    namespace: 'ci'
+    tag: 'centos'
+images:
```

Comment:
You can add the image source for `oc` in the build here

Reply:
the root image contains `oc` already.

Reply:
Right but the way you're installing it does not ensure that the `oc` that they use in the test is the same version as the cluster they are connecting to, which is bad.